File: src/main/java/ejerciciosJava/Ejercicio13.java
package ejerciciosJava;

class Anime {
    private final String nombre;
    private final int episodios;
    private final String genero;

    public Anime(String nombre, int episodios, String genero) {
        this.nombre = nombre;
        this.episodios = episodios;
        this.genero = genero;
    }

    @Override
    public String toString() {
        return "Nombre: " + nombre + "\nGénero: " + genero + "\nEpisodios: " + episodios;
    }
}

public class Ejercicio13 {
    public static void main() {
        Anime anime1 = new Anime("Hola", 33, "acción");
        Anime anime2 = new Anime("fsfags", 11, "suspense");

        System.out.println(anime2);
        System.out.println();
        System.out.println(anime1);
    }
}

File: src/main/java/ejerciciosKotlin/Ejercicio13.kt
package ejerciciosKotlin

class Anime(
    val nombre:String,
    val episodios:Int,
    val genero:String
){
    override fun toString(): String {
        return "Nombre: $nombre\nGénero: $genero\nEpisodios: $episodios"
    }
}

fun ejercicio13(){
    val anime1 = Anime("Hola", 33, "accion")
    val anime2 = Anime("fsfags", 11, "suspense")

    println(anime2)
    println()
    println(anime1)
}